src/utils/hooks/removeAccountDependenciesHooks.ts

```diff
@@ -25,3 +50,25 @@ export const useRemoveAccountsDependencies = () => {
     dispatch(assetsSlice.actions.removeAccountsData(pkhs));
   };
 };
+
+/**
+ * Hook for getting obsolete multisigs.
+ *
+ * I. e. multisigs for which all signer accounts were removed.
+ *
+ * Hook checks currently stored implicit accounts, but ignores deletedAccounts.
+ */
+const useGetMultisigsToRemove = () => {
+  const implicit = useImplicitAccounts();
+  const multisig = useMultisigAccounts();
+
+  return (deletedAccounts: ImplicitAccount[]) => {
```

Comment:
maybe it's simpler to pass in the remaining accounts? 
then the logic would be so simple you could inline it into the `useRemoveDependenciesAndMultisigs` definition

Reply:
I used accsToRemove patter here to reuse what was already implemented in `useRemoveAccountsDependencies` which also includes multiple slice actions & tests.

I agree, that this accsToRemain pattern is a better fit, but I'd rather do it separately to avoid having one massive PR: [asana link](https://app.asana.com/0/1205770721172203/1206826224836362/f)